l10n_it_central_journal/README.rst

```diff
@@ -13,7 +13,6 @@ Contributors
 ------------
 
 * Gianmarco Conte <gconte@dinamicheaziendali.it>
-* Lara Baggio <lbaggio@linkgroup.it>
```

Comment:
poor Lara, why?
Moreover you can generate a nice README using [this script](https://github.com/OCA/maintainer-tools/blob/master/tools/gen_addon_readme.py)